_docs/setup/kubernetes/quick-start.md

```diff
@@ -16,7 +16,7 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
 ## Prerequisites
 
 The following instructions require you have access to a Kubernetes **1.7.3 or newer** cluster
-with [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled. You will also need `kubectl` **1.7.3 or newer** installed.  If you wish to enable [automatic injection of sidecar]({{home}}/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection), you need to turn on Kubernetes alpha features in your cluster.
+with [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled. You will also need `kubectl` **1.7.3 or newer** installed.  If you wish to enable [automatic sidecar injection]({{home}}/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection), you need Kubernetes version 1.9 or greater.
 
```

Comment:
Jason, is 1.9 the only requirement?  Does it have to enable certain feature in 1.9?

Reply:
also, is there a requirement on kubectl version?

Reply:
Alpha gates must be enabled and runtime config of admissionregistration.k8s.io/v1beta1 must be enabled.

Reply:
The `admissionregistration.k8s.io/v1beta1` API and MutatingWebhook plugin do need to be enabled. My understanding is that this are enabled by default in 1.9 since the feature is now beta. Are beta features still guarded by alpha feature gate flags?

Reply:
No special requirement for kubectl beyond what we require for the rest of istio install.